README.md

```diff
@@ -2,44 +2,36 @@
 
 ## Package updates
 
-How to update the package from Arch on any system
+How to update the package from Arch on any system:
 
-0. Download the release tar.gz and run sha256sum on it.
-1. Edit PKGBUILD and change the `pkgver` and `sha256sums` fields.
-2. Run `mksrcinfo` (see below if you're not on Arch)
-3. Commit & push
+1. Edit the PKGBUILD and change the `pkgver` field.
+2. If you're on Arch, run `updpkgsums && makepkg --printsrcinfo > .SRCINFO`, and skip to the last step.
```

Comment:
```suggestion
2. If you're on Arch, run `./update`, and skip to the last step.
```
What do you think of wrapping those commands too?

Reply:
Now every method uses an update script.